demisto_sdk/commands/format/tests/test_formatting_md_test.py

```diff
@@ -1,32 +1,13 @@
-import filecmp
-import os
-import shutil
-
-import pytest
-
 from demisto_sdk.commands.format.update_description import DescriptionFormat
 from demisto_sdk.tests.constants_test import (
-    DESCRIPTION_PATH, DESTINATION_FORMAT_DESCRIPTION_COPY,
     SOURCE_DESCRIPTION_FORMATTED_CONTRIB_DETAILS,
-    SOURCE_DESCRIPTION_WITH_CONTRIB_DETAILS)
+    SOURCE_DESCRIPTION_FORMATTED_WITH_BETA_DESCRIPTION,
+    SOURCE_DESCRIPTION_WITH_CONTRIB_DETAILS,
+    SOURCE_DESCRIPTION_WITHOUT_BETA_DESCRIPTION)
 
 
 class TestDescriptionFormat:
-    @pytest.fixture(autouse=True)
-    def description_copy(self):
```

Comment:
why did you remove the fixtures?

Reply:
wanted both tests to have the same logic, and i cant use the same fixtures to both of the tests